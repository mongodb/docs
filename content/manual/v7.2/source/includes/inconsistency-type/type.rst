
Type of inconsistency document.  Returns a value of |incon-type| for this   
inconsistency type.


