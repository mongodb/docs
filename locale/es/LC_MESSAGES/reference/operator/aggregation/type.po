# SOME DESCRIPTIVE TITLE.
# Copyright (C) 2008-2016
# This file is distributed under the same license as the mongodb-manual
# package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: mongodb-manual 3.4\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2016-12-08 12:02-0500\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.6.0\n"

# d4bbf988f7974f2483b5728ffd72c3bf
#: ../source/reference/operator/aggregation/type.txt:3
msgid "$type (aggregation)"
msgstr ""

# b64bd8c1692c4eea99f174e663e70172
#: ../source/reference/operator/aggregation/type.txt
msgid "On this page"
msgstr ""

# e192009c6d8d4f9eb3dd791386a8525e
#: ../source/reference/operator/aggregation/type.txt:14
msgid "Definition"
msgstr ""

# 7d402f99fec8447ba373533442270be5
#: ../source/reference/operator/aggregation/type.txt:20
msgid ""
"Returns a string that specifies the :doc:`BSON type </reference/bson-"
"types/>` of the argument."
msgstr ""

# f945fd8c35f04e5aae6685fce1744a62
#: ../source/reference/operator/aggregation/type.txt:23
msgid ""
":expression:`$type` has the following :ref:`operator expression syntax "
"<agg-quick-ref-operator-expressions>`:"
msgstr ""

# 588f2206f51b43a5836b5161aa92cc0e
#: ../source/reference/operator/aggregation/type.txt:30
msgid "The argument can be any valid :ref:`expression <aggregation-expressions>`."
msgstr ""

# cc6ad411c27d468d97006b1a03f6cd20
#: ../source/reference/operator/aggregation/type.txt:34
msgid "Behavior"
msgstr ""

# b0cfa8014aac45ba812f016c4f0ae09c
#: ../source/reference/operator/aggregation/type.txt:36
msgid ""
"Unlike the :query:`$type` query operator, which matches array elements "
"based on their BSON type, the :expression:`$type` aggregation operator "
"does not examine array elements. Instead, when passed an array as its "
"argument, the ``$type`` aggregation operator returns the type of the "
"argument, i.e. ``\"array\"``."
msgstr ""

# 22f64d3123a3458a9400f3d2226e30b2
#: ../source/reference/operator/aggregation/type.txt:42
msgid ""
"If the argument is a field that is missing in the input document, "
":expression:`$type` returns the string ``\"missing\"``."
msgstr ""

# 3e4e567ed44a4eb99009492a9464a867
# bd517e8c1ac048f68984aa7b059b2a5e
#: ../source/reference/operator/aggregation/type.txt:53
#: ../source/reference/operator/aggregation/type.txt:89
msgid "Example"
msgstr ""

# 6bc0fb9c1be7482dbb2742b74c550a4d
#: ../source/reference/operator/aggregation/type.txt:54
msgid "Results"
msgstr ""

# ce53f805dce046ba9fadfbc807a17762
#: ../source/reference/operator/aggregation/type.txt:56
msgid "``{ $type: \"a\" }``"
msgstr ""

# eae4710260ac4f8e9ff0f2229752766c
#: ../source/reference/operator/aggregation/type.txt:57
msgid "``\"string\"``"
msgstr ""

# 66f62df1c9a14c01848db22f2c103136
#: ../source/reference/operator/aggregation/type.txt:59
msgid "``{ $type: /a/ }``"
msgstr ""

# 780556e0e1554dedbe1327da6ba07595
#: ../source/reference/operator/aggregation/type.txt:60
msgid "``\"regex\"``"
msgstr ""

# 4c60f0da177842f18367b5c7559507a6
#: ../source/reference/operator/aggregation/type.txt:62
msgid "``{ $type: 1 }``"
msgstr ""

# 1d3a927d71a94a06b4915840248d7a47
#: ../source/reference/operator/aggregation/type.txt:63
msgid "``\"double\"``"
msgstr ""

# fbd0acd7b10d45f4bcadf4b541747e2a
#: ../source/reference/operator/aggregation/type.txt:65
msgid "``{ $type: NumberLong(627) }``"
msgstr ""

# 0a5784fcb9e84d30b51cccab482a9a56
#: ../source/reference/operator/aggregation/type.txt:66
msgid "``\"long\"``"
msgstr ""

# 1f9af95d830c46c98a56437aafde2fe4
#: ../source/reference/operator/aggregation/type.txt:68
msgid "``{ $type: { x: 1 } }``"
msgstr ""

# d31c5584126f4fca8051223ac97ba04a
#: ../source/reference/operator/aggregation/type.txt:69
msgid "``\"object\"``"
msgstr ""

# 7246de5f63514059b431ce73400c951c
#: ../source/reference/operator/aggregation/type.txt:71
msgid "``{ $type: [ [ 1, 2, 3 ] ] }``"
msgstr ""

# b10cb37f321a45aca88ef60ed5c296ef
#: ../source/reference/operator/aggregation/type.txt:72
msgid "``\"array\"``"
msgstr ""

# eb2195ed901d4c3897c761dbca618177
#: ../source/reference/operator/aggregation/type.txt:76
msgid ""
"In the case of a literal array such as ``[ 1, 2, 3 ]``, enclose the "
"expression in an outer set of array brackets to prevent MongoDB from "
"parsing ``[ 1, 2, 3 ]`` as an :ref:`argument list <agg-quick-ref-"
"operator-expressions>` with three arguments (``1, 2, 3``). Wrapping the "
"array ``[ 1, 2, 3 ]`` in a :expression:`$literal` expression achieves the"
" same result."
msgstr ""

# 2eb5e505de2240d388cc7fdb1ac7adea
#: ../source/reference/operator/aggregation/type.txt:85
msgid ""
"See :ref:`operator expression syntax forms <agg-quick-ref-operator-"
"expressions>` for more information."
msgstr ""

# 4234818fa20a46328f7e67ebc05f2177
#: ../source/reference/operator/aggregation/type.txt:91
msgid ""
"This example uses a :term:`collection <collection>` named ``coll`` with "
"the following :term:`documents <document>`:"
msgstr ""

# c0eeac8b6795478988661a9c885d1db7
#: ../source/reference/operator/aggregation/type.txt:103
msgid ""
"The following aggregation operation uses the ``$type`` operator to "
"display the type of field ``a`` for all documents as part of the "
":pipeline:`$project` stage."
msgstr ""

# e5ac12ef5757452eb4e348cde61611df
#: ../source/reference/operator/aggregation/type.txt:115
msgid "The operation returns the following:"
msgstr ""

