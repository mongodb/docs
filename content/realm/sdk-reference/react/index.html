<!doctype html>
<head>
  <meta http-equiv="refresh" content="0; url=https://mongodb.com/docs/realm-sdks/react/latest" />
</head>
<body>
  <p><a href="https://mongodb.com/docs/realm-sdks/react/latest">Redirect</a></p>
</body>
