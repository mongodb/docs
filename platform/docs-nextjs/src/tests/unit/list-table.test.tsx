import { render } from '@testing-library/react';
import { matchers } from '@emotion/jest';
import ListTable from '@/components/list-table';

import mockData from '@/tests/data/list-table.test.json';
import mockDataFixedWidths from '@/tests/data/list-table-fixed-widths.test.json';
import type { ListTableNode } from '@/types/ast';

const typedMockData = mockData as ListTableNode;
const typedMockDataFixedWidths = mockDataFixedWidths as ListTableNode;

expect.extend(matchers);

const mountListTable = (data: ListTableNode) =>
  render(<ListTable nodeChildren={data.children} options={data.options} />);

describe('when rendering a list-table directive', () => {
  const data = typedMockData;

  it('renders correctly', () => {
    const wrapper = mountListTable(data);
    expect(wrapper.asFragment()).toMatchSnapshot();
  });

  it('displays one header row', () => {
    const wrapper = mountListTable(data);
    expect(wrapper.getByTestId('leafygreen-ui-header-row')).toBeTruthy();
  });

  it('displays six header columns', () => {
    const wrapper = mountListTable(data);
    expect(wrapper.queryAllByRole('columnheader')).toHaveLength(6);
  });

  it('displays five body rows', () => {
    const wrapper = mountListTable(data);
    // we test for 6 here, with an offset to account for the header row
    expect(wrapper.queryAllByRole('row')).toHaveLength(6);
  });

  it('renders one stub column per row', () => {
    const wrapper = mountListTable(data);
    expect(wrapper.queryAllByRole('rowheader')).toHaveLength(5);
  });
});

describe('when rendering a list table with fixed widths', () => {
  const data = typedMockDataFixedWidths;

  it('renders correctly', () => {
    const wrapper = mountListTable(data);
    expect(wrapper.asFragment()).toMatchSnapshot();
  });

  it('displays no header columns', () => {
    const wrapper = mountListTable(data);
    expect(wrapper.queryAllByRole('columnheader')).toHaveLength(0);
  });

  it('displays no header row when none are set', () => {
    const wrapper = mountListTable(data);
    expect(wrapper.queryByTestId('leafygreen-ui-header-row')).toBeFalsy();
    expect(wrapper.queryByText('name')).not.toBeTruthy();
  });

  it('displays no stub columns', () => {
    const wrapper = mountListTable(data);
    expect(wrapper.queryAllByRole('rowheader')).toHaveLength(0);
  });
});
