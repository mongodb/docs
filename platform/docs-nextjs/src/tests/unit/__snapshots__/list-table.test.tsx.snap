// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`when rendering a list table with fixed widths renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  overflow: auto;
  width: 100%;
  position: relative;
}

.emotion-1 {
  margin: 24px 0;
}

.emotion-2 {
  border-spacing: 0;
  border-collapse: collapse;
  width: 100%;
  color: #1C2D38;
  font-size: 13px;
  line-height: 20px;
}

.emotion-3 {
  background-color: #FFFFFF;
  box-shadow: 0 4px #E8EDEB;
}

.emotion-4 {
  position: relative;
  color: var(--font-color-primary);
  background-color: #FFFFFF;
  box-shadow: 0 4px #E8EDEB;
}

.dark-theme .emotion-4 {
  background-color: #001E2B;
  box-shadow: 0 4px #3D4F58;
}

.emotion-5 {
  padding: 10px 8px!important;
  vertical-align: top;
  color: var(--font-color-primary);
  overflow-wrap: anywhere;
  word-break: break-word;
  -webkit-align-content: flex-start;
  -ms-flex-line-pack: flex-start;
  align-content: flex-start;
}

.emotion-5 * {
  font-size: 13px!important;
}

.emotion-5>div {
  height: unset;
  min-height: unset;
}

.emotion-5>div {
  min-height: unset;
  max-height: unset;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
}

.emotion-6 {
  padding: 0 8px;
  overflow: hidden;
  padding-left: 32px;
}

.emotion-6:focus-visible {
  box-shadow: inset;
}

.emotion-6:last-child {
  padding-right: 24px;
}

.emotion-7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  min-height: 40px;
  transition-property: min-height,max-height,opacity,padding,transform;
  transition-duration: 150ms;
  transition-timing-function: ease;
  opacity: 1;
  min-height: 40px;
  max-height: 40px;
  -webkit-box-pack: left;
  -ms-flex-pack: left;
  -webkit-justify-content: left;
  justify-content: left;
  text-align: left;
}

.emotion-8 *,
.emotion-8 p,
.emotion-8 a {
  line-height: 20px;
}

.emotion-8>*,
.emotion-8>div p {
  margin: 0 0 12px!important;
}

.emotion-8>div *:last-child,
.emotion-8>*:last-child {
  margin-bottom: 0!important;
}

.emotion-9 {
  margin: unset;
  font-family: 'Euclid Circular A','Helvetica Neue',Helvetica,Arial,sans-serif;
  color: #001E2B;
  font-size: 13px;
  line-height: 20px;
  color: #001E2B;
  font-weight: 400;
  margin-bottom: 16px;
  color: var(--font-color-primary);
}

.emotion-9 strong,
.emotion-9 b {
  font-weight: 600;
}

.emotion-10 {
  padding: 0 8px;
  overflow: hidden;
}

.emotion-10:focus-visible {
  box-shadow: inset;
}

.emotion-10:last-child {
  padding-right: 24px;
}

<div
    class="emotion-0 emotion-1"
    tabindex="0"
  >
    <table
      class="emotion-2"
      data-lgid="lg-table"
    >
      <colgroup>
        <col
          style="width: 5%;"
        />
        <col
          style="width: 10%;"
        />
        <col
          style="width: 15%;"
        />
        <col
          style="width: 20%;"
        />
        <col
          style="width: 30%;"
        />
        <col
          style="width: 20%;"
        />
      </colgroup>
      <thead
        class="emotion-3 emotion-4"
      />
      <tbody>
        <tr
          aria-hidden="false"
          class=""
          data-expanded="false"
          data-lgid="lg-table-row"
          data-selected="false"
          id="lg-table-row-0"
        >
          <td
            class="emotion-5 emotion-6"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-7"
              data-state="entered"
            >
              <div
                class="emotion-8"
              >
                <p
                  class="emotion-9"
                >
                  notebook
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-10 emotion-5"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-7"
              data-state="entered"
            >
              <div
                class="emotion-8"
              >
                <p
                  class="emotion-9"
                >
                  50
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-10 emotion-5"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-7"
              data-state="entered"
            >
              <div
                class="emotion-8"
              >
                <p
                  class="emotion-9"
                >
                  8.5x11,in
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-10 emotion-5"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-7"
              data-state="entered"
            >
              <div
                class="emotion-8"
              >
                <p
                  class="emotion-9"
                >
                  A
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-10 emotion-5"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-7"
              data-state="entered"
            >
              <div
                class="emotion-8"
              >
                <p
                  class="emotion-9"
                >
                  college-ruled,perforated
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-10 emotion-5"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-7"
              data-state="entered"
            >
              <div
                class="emotion-8"
              >
                <p
                  class="emotion-9"
                >
                  8
                </p>
              </div>
            </div>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</DocumentFragment>
`;

exports[`when rendering a list-table directive renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  overflow: auto;
  width: 100%;
  position: relative;
}

.emotion-1 {
  margin: 24px 0;
}

.emotion-2 {
  border-spacing: 0;
  border-collapse: collapse;
  width: 100%;
  color: #1C2D38;
  font-size: 13px;
  line-height: 20px;
}

.emotion-3 {
  background-color: #FFFFFF;
  box-shadow: 0 4px #E8EDEB;
}

.emotion-4 {
  position: relative;
  color: var(--font-color-primary);
  background-color: #FFFFFF;
  box-shadow: 0 4px #E8EDEB;
}

.dark-theme .emotion-4 {
  background-color: #001E2B;
  box-shadow: 0 4px #3D4F58;
}

.emotion-5 {
  padding: 10px 8px!important;
  vertical-align: top;
  color: var(--font-color-primary);
  line-height: 24px;
  font-weight: 600;
  font-size: 13px;
  width: auto;
}

.emotion-5 * {
  font-size: 13px!important;
}

.emotion-5>div {
  height: unset;
  min-height: unset;
}

.emotion-6 {
  padding: 0 8px;
  overflow: hidden;
  padding-left: 32px;
  width: 150px;
}

.emotion-6:focus-visible {
  box-shadow: inset;
}

.emotion-6:last-child {
  padding-right: 24px;
}

.emotion-7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  min-height: 40px;
  transition-property: min-height,max-height,opacity,padding,transform;
  transition-duration: 150ms;
  transition-timing-function: ease;
  height: 40px;
  -webkit-box-pack: left;
  -ms-flex-pack: left;
  -webkit-justify-content: left;
  justify-content: left;
  text-align: left;
}

.emotion-9 {
  padding: 0 8px;
  overflow: hidden;
  width: 150px;
}

.emotion-9:focus-visible {
  box-shadow: inset;
}

.emotion-9:last-child {
  padding-right: 24px;
}

.emotion-23:nth-of-type(even) {
  background-color: #F9FBFA;
}

.emotion-24:nth-of-type(even) {
  background-color: #F9FBFA;
}

.dark-theme .emotion-24:nth-of-type(even) {
  background-color: #112733;
}

.emotion-25 {
  padding: 10px 8px!important;
  vertical-align: top;
  color: var(--font-color-primary);
  overflow-wrap: anywhere;
  word-break: break-word;
  -webkit-align-content: flex-start;
  -ms-flex-line-pack: flex-start;
  align-content: flex-start;
  background-color: #F9FBFA;
  border-right: 3px solid #E8EDEB;
  font-weight: 600;
}

.emotion-25 * {
  font-size: 13px!important;
}

.emotion-25>div {
  height: unset;
  min-height: unset;
}

.emotion-25>div {
  min-height: unset;
  max-height: unset;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
}

.dark-theme .emotion-25 {
  background-color: #112733;
  border-right: 3px solid #3D4F58;
}

.emotion-26 {
  padding: 0 8px;
  overflow: hidden;
  padding-left: 32px;
}

.emotion-26:focus-visible {
  box-shadow: inset;
}

.emotion-26:last-child {
  padding-right: 24px;
}

.emotion-27 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  min-height: 40px;
  transition-property: min-height,max-height,opacity,padding,transform;
  transition-duration: 150ms;
  transition-timing-function: ease;
  opacity: 1;
  min-height: 40px;
  max-height: 40px;
  -webkit-box-pack: left;
  -ms-flex-pack: left;
  -webkit-justify-content: left;
  justify-content: left;
  text-align: left;
}

.emotion-28 *,
.emotion-28 p,
.emotion-28 a {
  line-height: 20px;
}

.emotion-28>*,
.emotion-28>div p {
  margin: 0 0 12px!important;
}

.emotion-28>div *:last-child,
.emotion-28>*:last-child {
  margin-bottom: 0!important;
}

.emotion-29 {
  margin: unset;
  font-family: 'Euclid Circular A','Helvetica Neue',Helvetica,Arial,sans-serif;
  color: #001E2B;
  font-size: 13px;
  line-height: 20px;
  color: #001E2B;
  font-weight: 400;
  margin-bottom: 16px;
  color: var(--font-color-primary);
}

.emotion-29 strong,
.emotion-29 b {
  font-weight: 600;
}

.emotion-30 {
  padding: 0 8px;
  overflow: hidden;
}

.emotion-30:focus-visible {
  box-shadow: inset;
}

.emotion-30:last-child {
  padding-right: 24px;
}

.emotion-31 {
  padding: 10px 8px!important;
  vertical-align: top;
  color: var(--font-color-primary);
  overflow-wrap: anywhere;
  word-break: break-word;
  -webkit-align-content: flex-start;
  -ms-flex-line-pack: flex-start;
  align-content: flex-start;
}

.emotion-31 * {
  font-size: 13px!important;
}

.emotion-31>div {
  height: unset;
  min-height: unset;
}

.emotion-31>div {
  min-height: unset;
  max-height: unset;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
}

<div
    class="emotion-0 emotion-1"
    tabindex="0"
  >
    <table
      class="emotion-2"
      data-lgid="lg-table"
    >
      <thead
        class="emotion-3 emotion-4"
      >
        <tr
          data-testid="leafygreen-ui-header-row"
        >
          <th
            class="emotion-5 emotion-6"
            data-lgid="lg-table-header"
            scope="col"
          >
            <div
              class="emotion-7"
            >
              <div>
                name
              </div>
            </div>
          </th>
          <th
            class="emotion-5 emotion-9"
            data-lgid="lg-table-header"
            scope="col"
          >
            <div
              class="emotion-7"
            >
              <div>
                quantity
              </div>
            </div>
          </th>
          <th
            class="emotion-5 emotion-9"
            data-lgid="lg-table-header"
            scope="col"
          >
            <div
              class="emotion-7"
            >
              <div>
                size
              </div>
            </div>
          </th>
          <th
            class="emotion-5 emotion-9"
            data-lgid="lg-table-header"
            scope="col"
          >
            <div
              class="emotion-7"
            >
              <div>
                status
              </div>
            </div>
          </th>
          <th
            class="emotion-5 emotion-9"
            data-lgid="lg-table-header"
            scope="col"
          >
            <div
              class="emotion-7"
            >
              <div>
                tags
              </div>
            </div>
          </th>
          <th
            class="emotion-5 emotion-9"
            data-lgid="lg-table-header"
            scope="col"
          >
            <div
              class="emotion-7"
            >
              <div>
                rating
              </div>
            </div>
          </th>
        </tr>
      </thead>
      <tbody>
        <tr
          aria-hidden="false"
          class="emotion-23 emotion-24"
          data-expanded="false"
          data-lgid="lg-table-row"
          data-selected="false"
          id="lg-table-row-0"
        >
          <td
            class="emotion-25 emotion-26"
            data-lgid="lg-table-cell"
            role="rowheader"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  journal
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  25
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  14x21,cm
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  A
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  brown, lined
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  9
                </p>
              </div>
            </div>
          </td>
        </tr>
        <tr
          aria-hidden="false"
          class="emotion-23 emotion-24"
          data-expanded="false"
          data-lgid="lg-table-row"
          data-selected="false"
          id="lg-table-row-1"
        >
          <td
            class="emotion-25 emotion-26"
            data-lgid="lg-table-cell"
            role="rowheader"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  notebook
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  50
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  8.5x11,in
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  A
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  college-ruled,perforated
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  8
                </p>
              </div>
            </div>
          </td>
        </tr>
        <tr
          aria-hidden="false"
          class="emotion-23 emotion-24"
          data-expanded="false"
          data-lgid="lg-table-row"
          data-selected="false"
          id="lg-table-row-2"
        >
          <td
            class="emotion-25 emotion-26"
            data-lgid="lg-table-cell"
            role="rowheader"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  paper
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  100
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  8.5x11,in
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  D
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  watercolor
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  10
                </p>
              </div>
            </div>
          </td>
        </tr>
        <tr
          aria-hidden="false"
          class="emotion-23 emotion-24"
          data-expanded="false"
          data-lgid="lg-table-row"
          data-selected="false"
          id="lg-table-row-3"
        >
          <td
            class="emotion-25 emotion-26"
            data-lgid="lg-table-cell"
            role="rowheader"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  planner
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  75
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  22.85x30,cm
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  D
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  2019
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  10
                </p>
              </div>
            </div>
          </td>
        </tr>
        <tr
          aria-hidden="false"
          class="emotion-23 emotion-24"
          data-expanded="false"
          data-lgid="lg-table-row"
          data-selected="false"
          id="lg-table-row-4"
        >
          <td
            class="emotion-25 emotion-26"
            data-lgid="lg-table-cell"
            role="rowheader"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  postcard
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  45
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  10x,cm
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  D
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  double-sided,white
                </p>
              </div>
            </div>
          </td>
          <td
            class="emotion-30 emotion-31"
            data-lgid="lg-table-cell"
          >
            <div
              class="emotion-27"
              data-state="entered"
            >
              <div
                class="emotion-28"
              >
                <p
                  class="emotion-29"
                >
                  2
                </p>
              </div>
            </div>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</DocumentFragment>
`;
